content/docs/user-guide/assets/pipeline/user_defined_properties.md

```diff
@@ -0,0 +1,123 @@
+
+The O3DE scene pipeline reads in this UDP metadata and added to the scene graph as a node with CustomPropertyData contents. This UDP metadata will do nothing itself since it is meant to be properties that other scene building logic units will access in order to perform build operations and/or modifications.
+
+Inside the CustomPropertyData there is a PropertyMap that stores a dictionary of string-to-value properties for that node. The scene builders will be able to access this metadata using its PropertyMap. The properties can store value types of string, Boolean, unsigned integer 32-bit, signed integer 64-bit, float, and double. The metadata can be used to modify the scene’s rule manifest, parameterize scene product assets, and set scene parameters downstream. Each node can have a PropertyMap where the keys do not repeat. The PropertyMap is read only like the other elements of the scene graph.
+
+The CustomPropertyData can be accessed by scene builders using either the Python or C++ API by handling any event that takes the scene graph as input. The typical method is to handle the Update Manifest event so that the scene builder can modify the scene manifest rules.
+
+## Assigning Materials in Default Procedural Prefab using UDPs
+
+The default procedural prefab scene builder inside the Prefab game has an example of how UDP metadata can be used to customized the scene pipeline. The Prefab gem constructs a default prefab for a scene graph, it looks for the UDP ```o3de.default.material``` in a mesh data node to define an O3DE render material for that mesh group inside the prefab. This allows artists to assign O3DE render materials back in the DCC tool. Each time the source scene file (i.e. FBX) is exported, this reference is maintained through the scene pipeline.
+
+## Future Use Cases of UDP metadata
+
+There are many use cases for UDP metadata in both the O3DE engine and for content teams in the future. For any soft naming rule such as naming nodes with a ```_lod0``` to indicate level of detail (LOD), a user defined property key can be used instead.
+
+These are a few features that will be managed using user defined properties in the future.
+
+### Level of Detail
+
+Instead of using a soft naming format to discover LODs like ```lod_<n>```, an artist will be able to add a property named ```o3de.default.lod``` and assign it a number from 0 (zero – highest level of detail) to 4 (four – lowest level of detail) to a mesh node. This will make it possible to group mesh nodes per LOD in the DCC tool.
```

Comment:
Are we sure we're going to remove soft naming for LODs? I imagine it will be an "in addition to" and not "instead of" situation.